csfieldguide/chapters/models.py

```diff
@@ -32,8 +33,9 @@ class Chapter(models.Model):
     slug = models.SlugField(unique=True)
     name = models.CharField(max_length=100)
     number = models.SmallIntegerField(unique=True)
+    introduction = models.TextField()
     other_resources = models.TextField(null=True)
```

Comment:
What is `other-resources` for a chapter? This seems to be a duplicate of the "Further information" section of a chapter. I don't think this attribute is required for a chapter but keen to hear your thoughts?